### PR TITLE
Onboard Magic Link feature into the Console app

### PR DESCRIPTION
**Pull request by thivi**

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/login.jsp [hunk 1]
```diff
@@ -387,6 +388,26 @@
                                     </span>
                                 </button>
                             </div>
+                            <%
+                                        }
```
Comment (brionmario) [inline apps/authentication-portal/src/main/webapp/login.jsp:392]: Check if there's a formatting issue here.

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx [hunk 2]
```diff
@@ -137,17 +137,22 @@ export const AuthenticationStep: FunctionComponent<AuthenticationStepPropsInterf
...
+                IdentityProviderManagementConstants.IDENTIFIER_FIRST_AUTHENTICATOR ]
+                .includes(option.authenticator)) {
+                setShowSubjectIdentifierCheckbox(false);
+            } else {
+                setShowSubjectIdentifierCheckbox(true);
+            }
         }
-    }
-    );
+        );
```
Comment (brionmario) [inline apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx:154]:
```
            } else {
                setShowSubjectIdentifierCheckbox(true);
            }
        });
```

**Review by brionmario — APPROVED** [review]

File: apps/authentication-portal/src/main/webapp/login.jsp (hunk 1 above)
Comment (thivi) [reply apps/authentication-portal/src/main/webapp/login.jsp]: Resolved by [85e6583](https://github.com/wso2/identity-apps/pull/2989/commits/85e6583736ad5d36a0e9c8949caf1c73b2b2be39)

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx (hunk 2 above)
Comment (thivi) [reply apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx]: Resolved by [85e6583](https://github.com/wso2/identity-apps/pull/2989/commits/85e6583736ad5d36a0e9c8949caf1c73b2b2be39)

**Pull request merged**